ui/installer/VCSA/install.sh

```diff
@@ -36,14 +39,10 @@ while IFS='' read -r line; do
     eval $line
 done < ./configs
 
-# check for the VC IP
-if [[ $VCENTER_IP == "" ]] ; then
-    echo "Error! vCenter IP cannot be empty. Please provide a valid IP in the configs file"
-    cleanup
-    exit 1
-fi
+# replace space delimiters with colon delimiters 
```

Comment:
The thumbprint set into configs is already in this format:
`"4E:A2:37:C4:DF:B3:1A:EE:DE:94:70:79:60:5E:DC:CF:34:8F:B9:83"`